### ISSUE TITLE
How to get the best model for evaluating in another test set?

### ISSUE THREAD

**Issue by chenqi008** [twtygqyy/pytorch-SRDenseNet#6]
Hi,

I just want to know how to get the best model from the checkpoint models. Because I have trained a series of checkpoints and use the last one checkpoint models (model_epoch_60.pth) to evaluate in the Set5 dataset. However, I can not achieve the performance as shown in this code.

Thank you very much!

**Comment by twtygqyy** [comment]
Hi @chenqi008, I used the default settings in my code. What kind of dataset did you applied for training? Also, could you show me the performance of the model you trained? 
There is lots of randomness for training a model.

**Comment by chenqi008** [comment]
Hi @twtygqyy, I used 10k images randomly selected from the imagenet dataset to train this model. I also use the default settings in this code. But it can only achieve about 30 in Set5. Are the iterations of training not enough?

**Comment by twtygqyy** [comment]
Hi @chenqi008, normally I used the last check point, there might be some variations among the check points before you finish the training, but only slightly difference. 
About the performance, did you use the dataset generation code in the repo? In my experiment, 30,000 images with data augmentations including flipping, rotation, downsizing are adopted for training. I guess this leads to the performance gap.

**Comment by chenqi008** [comment]
Hi @twtygqyy, thanks a lot. I get a promising result when I increase the learning epochs in training.
